docs/pages/branding/about.tsx

```diff
@@ -170,6 +173,28 @@ const Person = (props: Profile & { sx?: PaperProps['sx'] }) => {
   );
 };
 
+const Widget = ({
```

Comment:
What's a "Widget" here? That term is fairly generic.

Reply:
The point is to reduce some duplication in the code and make it generic. I don't have a better name since it is only used in this file.